addons/delivery/models/sale_order.py

```diff
@@ -91,12 +91,9 @@ def action_open_delivery_wizard(self):
             }
         }
 
-    def _create_delivery_line(self, carrier, price_unit):
-        SaleOrderLine = self.env['sale.order.line']
-        context = {}
+    def _prepare_delivery_line_vals(self, carrier, price_unit):
         if self.partner_id:
             # set delivery detail in the customer language
-            context['lang'] = self.partner_id.lang
```

Comment:
I forgot, put it back please.

Reply:
From what I know of it, it is *implicitly* used by the translation engine (`_(...)`) below.

Reply:
ok thanks. You are right.